index.html

```diff
@@ -298,41 +301,38 @@
 
 
   d3.select("#sort-asc")
-      .on("click", function(){
-        d3.selectAll('#sorting-buttons button').classed('active', false);
-         
-          if (options.sorted === 'asc') {
-            options.sorted === false;
-          } else {
-            options.sorted ==='asc';
-
-          d3.select('#sort-asc').classed('active',true);
-          }
+      .on("click", function(d){
+           	 
+		    if (options.sorted === 'asc') {
```

Comment:
I don't think I'm referencing options.sorted correctly, but the buttons seem to be behaving closer to what I would expect. But there's no sorting.